src/auth-service/services/mailer.js

```diff
@@ -17,6 +17,8 @@ const mailer = {
 
       const mailOptions = {
         from: constants.EMAIL,
+        name: "AirQo Data Team",
```

Comment:
adding a sender name to the email in addition to making configurations in Gmail (See attached screenshot)
Unfortunately, this is not bringing the desired change we need., Anyway, let it be in as we continue to investigate. 
<img width="682" alt="Screenshot 2022-09-25 at 18 52 21" src="https://user-images.githubusercontent.com/1590213/192155099-b92cb9d5-b36a-43ed-8ca7-728acd9cdb5f.png">

Reply:
Thanks for this @Baalmart. 

I think the desired change was to have a professional name for the _data@airqo.net_ email responses. I can see that has been changed. Was there supposed to be another outcome?

Otherwise I see a need to have the **name** attribute in the mail options as a variable, so that it's changed once rather than manually changing it everywhere it appears in the file.

Reply:
Thanks @FaithDaka for this. I guess we can definitely make it an ENV. Shall update PR asap.

As for the change, appears it is not "working" after many tests. There is some background magic that could be beyond our control!

Perhaps we just need to do a little more research on this. 🤔